### ISSUE TITLE
Outdated videos

### ISSUE THREAD

**Issue by Zaliphyr** [githubtraining/introduction-to-github#73]
A couple of the YouTube links will only send you to private videos. 

Video: Exploring a repository - www.youtube.com/watch?v=R8OAwrcMlRw
Using GitHub Issues - www.youtube.com/watch?v=Zhj46r5D0nQ
Video: Watching, notifications, stars, and explore - www.youtube.com/watch?v=ocQldxF7fMY

![Bilde_2021-07-31_212252](https://user-images.githubusercontent.com/10731609/127750433-98f845a9-1d95-4533-80ef-5ddc23561aab.png)

**Comment by Zaliphyr** [comment]
After getting deeper into the tutorials, I find that most of the videos are outdated, making the whole tutorial really hard to follow. 

![Bilde_2021-07-31_213753](https://user-images.githubusercontent.com/10731609/127750727-e707183d-eabb-41de-99df-a61def46093d.png)

This refers to the video, witch I cant see as its private.

**Comment by danpoynor** [comment]
I can confirm this bug.

Comments on a post in the GitHub Support Community area shows a couple other folks are experiencing this also https://github.community/t/some-learning-lab-youtube-videos-are-marked-as-private-un-viewable/193177

**Comment by danpoynor** [comment]
These are working for me now. My understanding is a YouTube privacy policy change caught some videos that needed settings updated.

I think this issue can be closed.

**Comment by brianamarie** [comment]
Thank you both, @lizardion1 @danpoynor! I'm glad the videos are now accessible.
